pkgs/development/haskell-modules/configuration-nix.nix

```
@@ -535,21 +535,20 @@ self: super: builtins.intersectAttrs super {
       })
       (addBuildTools (with pkgs.buildPackages; [makeWrapper python3Packages.sphinx]) super.futhark);
 
-  git-annex = let
-    pathForDarwin = pkgs.lib.makeBinPath [ pkgs.coreutils ];
-  in overrideCabal (drv: pkgs.lib.optionalAttrs (!pkgs.stdenv.isLinux) {
+  git-annex = overrideCabal (drv: pkgs.lib.optionalAttrs (!pkgs.stdenv.isLinux) {
```

Your intention is to apply this everywhere, right?

```suggestion
  git-annex = overrideCabal (drv: {
```